src/attestation/README.md

```diff
@@ -1,122 +1,74 @@
 Microsoft Azure CLI 'attestation' Extension
 ==========================================
 
-This package is for the 'attestation' extension, i.e. `az attestation`.
-More info on [Microsoft Azure cloud security attestation](https://azure.microsoft.com/en-us/blog/microsoft-azure-updates-cloud-security-attestation/).
-
-### How to use ###
+### Usage ###
+#### Install the extension ####
 Install this extension using the below CLI command
 ```
 az extension add --name attestation
 ```
-To see command arguments details, should run the command with `-h`.
-Example:
-```
-az attestation create -h
-```
-```
-az attestation list -h
-```
-```
-az attestation show -h
-```
-```
-az attestation delete -h
-```
-
-### Included features ###
-#### Create an attestation ####
-The parameter `certs_input_path` is a path to your certificates pem file, it conforms to x5c in [RFC7517](https://tools.ietf.org/html/rfc7517#section-4.7).
-
-Example:
+#### Check the version ####
 ```
-az attestation create \
---location "eastus2" \
---provider-name "myattestationprovider" \
---resource-group "MyResourceGroup"
+az extension show --name attestation --query version
 ```
-Notice:
-May not all the values from `az account list-locations` are supported to create an attetation right now, more regions will be added in the future.
-
-#### List all attestations ####
-Example:
-List all attestations in a subscription
+#### Connect to Azure subscription ####
 ```
-az attestation list
+az login
+az account set -s {subs_id}
 ```
-List all attestations in a resource group
+#### Create a resource group (or use an existing one) ####
 ```
-az attestation list \
---resource-group "MyResourceGroup"
+az group create -n testrg -l westus
 ```
-
-#### Show the status of one attestation ####
-Example:
+#### Create provider in AAD mode ####
 ```
-az attestation show \
---provider-name "myattestationprovider" \
---resource-group "MyResourceGroup"
+az attestation create -n testatt1 -g testrg -l westus
 ```
-
-#### Delete an attestation ####
-Example:
+#### Get default policy ####
 ```
-az attestation delete \
---name "myattestationprovider" \
---resource-group "MyResourceGroup"
+az attestation policy show -n testatt1 -g testrg --attestation-type SGX-IntelSDK
 ```
+#### Configure policy in Text format using file path ###
+Download the policy file: https://github.com/Azure/azure-cli-extensions/blob/master/src/attestation/azext_attestation/tests/latest/policies/text_sgx_policy.txt
 
-#### Add a new attestation policy certificate ####
-Example:
+Content:
 ```
-az attestation signer add \
--n "myattestationprovider" -g "MyResourceGroup" \
---signer "eyAiYWxnIjoiUlMyNTYiLCAie..."
+version= 1.0;
+authorizationrules {
+[ type=="$is-debuggable", value==false ]
+&& [ type=="$product-id", value==4639 ]
+&& [ type=="$min-svn", value>= 0 ]
+&& [ type=="$sgx-mrsigner", value=="E31C9E505F37A58DE09335075FC8591254313EB20BB1A27E5443CC450B6E33E5"]
```

Comment:
this is fake value right

Reply:
@yungezz I don't know the meaning of these fields but I think it's not a fake value. However, the data was obtained from the service team and the PM wants to use this data in the instruction script.

Reply:
The PM also agreed that it's ok to use Github README.md to be the format of the instruction script.